package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "aurelia-Pager",
+  "version": "0.0.1",
+  "description": "Aurelia-only pagination",
+  "bugs": {
+    "url": "https://github.com/SpoonX/aurelia-Pager/issues"
+  },
+  "keywords": [
+    "aurelia",
+    "pagination",
+    "pager",
+    "spoonx"
+  ],
+  "author": "VMBindraban <vijay@spoonx.nl>",
```

Comment:
SpoonX

Author is iets waar op gezocht kan worden. Je kan een kopje contributors toevoegen :)
https://docs.npmjs.com/files/package.json#people-fields-author-contributors